src/main/java/formflow/library/inputs/FieldDesignation.java

```diff
@@ -0,0 +1,14 @@
+package formflow.library.inputs;
+
+/**
+ * A class to capture any string constants that might apply to fields in the inputs class.
+ */
+public class FieldDesignation {
+
+  /**
+   * A string used to indicate if an input field is a dynamic field. If this string is found in a field's name in a form
+   * submission, then it's likely the field is a dynamic field.
+   */
+  public static final String DYNAMIC_FIELD_MARKER = "_wildcard_";
```

Comment:
@bseeger what's the difference between this file and `UnvalidatedField.java`? Both are a class with defined strings in them that some fields might have. Though I think the dynamic fields still want to go through validation service.

Can the two files be combined?

Reply:
Yes, I think they totally could.  The naming of `UnvalidatedField` doesn't seem to encompass what we're trying to do here, since the dynamic fields are validated, just that the marker and trailing suffix are taken off first.   I'll tinker with naming and see what I can come up with for this.  Let me know if you have an ideas.

Reply:
I changed this around - let me know what you think @coltborg